unison-src/transcripts/delete.output.md

```diff
@@ -106,17 +94,7 @@ A delete should remove both versions of the term.
 ```ucm
 .a> delete foo
```

Comment:
Might be worth changing some of these transcript usages to verbose deletes, having the diff output seems useful in transcripts in some cases.